### ISSUE TITLE
Blender 2.8 support

### ISSUE THREAD

**Issue by machinekoder** [KDAB/kuesa-blender#9]
The next major Blender release 2.8 breaks the plugin APIs.

Blender 2.8 is so different from the user interface and rendering features to Blender 2.6, that is recommended to start with Blender 2.8 right away for newcomers.

I've tried to install the add-ons, but unfortunately, I hit some errors:
1. There is already glTF 2.0 export plugin in Blender 2.8 with the same name.
2. When trying to load the Kuesa plugin, I get a "Plugin has not been upgraded for Blender 2.8" error.

![image](https://user-images.githubusercontent.com/1467368/55479602-bd830a00-561e-11e9-90b2-9731fa217408.png)

**Comment by lemirep** [comment]
Thanks for the report. We've indeed only been using Blender 2.79 up until now. We will investigate changes required to make it work on 2.8.

**Comment by machinekoder** [comment]
In the meantime, do standard glTF 2.0 export files work with Kuesa?

**Comment by mkrus** [comment]
yes, Kuesa mostly expects default glTF. Exporters are only really useful if you need to use one of our custom extensions such as being able to group objects in layers
